src/extraction/wikidata.py

```diff
@@ -229,20 +235,33 @@ def _write_to_db(
     db_conn: sqlite3.Connection,
     title_to_id: Dict[str, str],
     id_to_attrs: Dict[str, Dict[str, Any]],
+    lang: str,
 ) -> None:
     """Persists qid information to database.
     db_conn (Connection): Database connection.
     title_to_id (Dict[str, str]): Titles to QIDs.
     id_to_attrs (Dict[str, Dict[str, Any]]): For QID a dictionary with property name to property value(s).
+    lang (str): Language.
     """
 
-    entities: List[Tuple[Optional[str], ...]] = []
+    entities: List[Tuple[str, int]] = []
     entities_texts: List[Tuple[Optional[str], ...]] = []
     aliases_for_entities: List[Tuple[str, str, int]] = []
+    # Text snippets indicating entity is a meta entity, i.e. a category or disambiguation page.
+    with open(
+        Path(__file__).parent.parent.parent / "configs" / "meta_terms.yaml", "r"
+    ) as stream:
+        meta_indicators = set(yaml.safe_load(stream)["entities"][lang])
```

Comment:
What does this do when `["entities"][lang]` is not an entry in the `.yaml` for a given `lang` that is not `en` or `es`?

Reply:
Crash. What it should do is to return an empty `Collection`. Will be fixed.

Reply:
Fixed in https://github.com/explosion/wikid/pull/16/commits/bdc65758180bd7967b4c975ef5f30cc1337ca75c.